CHANGELOG

```diff
@@ -4,8 +4,10 @@ v3.17 (Month 2020)
   * Card improvements:
     - Activity Feed now shows board name and link
     - No mandatory due date
-  * Comments can now have Textile markup
-  * Comments now show author's name instead of email
+  * Comments
+    - Added auto-save JS that uses localStorage on comment textarea
```

Comment:
```suggestion
    - Added auto-save JS that uses localStorage on comment textarea
```
^ This is developer speak, how do you explain this in a way that the user understands the change?